File: 0153-find-minimum-in-rotated-sorted-array/0153-find-minimum-in-rotated-sorted-array.cpp
class Solution {
public:
    int findMin(vector<int>& nums) {
        int min = nums[0];
        for(int i = 0; i < nums.size() -1; i++) {
            if(nums[i] < nums[i+1]) {
                continue;
            }
            else {
                min = nums[i+1];
                break;
            }
        }

        return  min;
    }
};

File: 0153-find-minimum-in-rotated-sorted-array/0153-find-minimum-in-rotated-sorted-array.java
class Solution {
    public int findMin(int[] nums) {
        int min = nums[0];

        for(int i = 0; i < nums.length -1 ; i++) {
            if( nums[i] < nums[i+1]) {
                continue;
            }
            else {
                min  = nums[i+1];
            }
        }
        return min;
    }
}